utils/remote_control/navigator_emergency_control/nodes/nodes/navigator_emergency.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+
+from remote_control_lib import RemoteControl
+import rospy
+from sensor_msgs.msg import Joy
+
+
+__maintainer__ = "David Zobel"
+__email__ = "zobeldavid@ufl.edu"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node("emergency")
+
+
+class Joystick(object):
+
+    def __init__(self):
+        self.force_scale = rospy.get_param("~force_scale", 600)
+        self.torque_scale = rospy.get_param("~torque_scale", 500)
```

Comment:
Oh thanks, I had thought it would grab these parameters from /joystick_wrench with this notation. I guess it only worked because it used the default value.